### ISSUE TITLE
[iOS] Xcode 8.3 / Swift 3 compatability

### ISSUE THREAD

**Issue by Pr0duktiv** [numenta/numenta-apps#922]
It is not possible to compile the application with the current Xcode version due to the abandonment of support for Swift 2.3.
Are there any endeavours to port this application to Swift 3? I'd be happy to contribute

**Comment by Pr0duktiv** [comment]
Okay I have been working on this and I managed to convert most of the code to Swift 3, still working on the AWS sections. I ran into an issue though:

`AccessDeniedException: Message=User:[...] is not authorized to perform: dynamodb:Scan on resource: arn:aws:dynamodb:[...]:table/taurus.metric.production})`

Is there any way of getting access to this? I forked the repo and would like to make it work before making a pull request.
